server/templates/staff/sidebar.html

```diff
@@ -20,7 +20,7 @@
             {% for course in courses %}
               <li class="treeview  {% if (request.endpoint == "admin.index" and loop.length == 1) %}active default{% endif %} {% if request.view_args['cid'] == course.id %} active{% endif %}" >
                 <a href="#">
-                  <i class="fa fa-university"></i> <span>{{ course.display_name_with_semester }}</span>
+                  <i class="fa fa-university"></i> <span>{{ course.display_name_with_semester | truncate(25, True) }}</span>
```

Comment:
![image](https://cloud.githubusercontent.com/assets/882381/17993816/2d441e10-6b08-11e6-895b-2d5e1b475401.png)
I tried text-overflow for a bit and it didn't do what I wanted - so I'm going to stick with this.